recipes/opentdf-client/all/conanfile.py

```diff
@@ -60,6 +64,9 @@ def validate(self):
             if Version(self.settings.compiler.version) < min_version:
                 raise ConanInvalidConfiguration(f"{self.name} requires C++{self._minimum_cpp_standard} support."
                                                  "The current compiler {self.settings.compiler} {self.settings.compiler.version} does not support it.")
+        if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
+            if self.settings.compiler.runtime == "MT" or self.settings.compiler.runtime == "MTd":
```

Comment:
There's an `is_msvc` help since there's the new compiler, we also have a helper for static runtimes https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc-static-runtime

Reply:
Fixed